proto/vulcanize/registry/v1beta1/attributes.proto

```diff
@@ -30,4 +32,172 @@ message WebsiteRegistrationRecord {
   string tls_cert_cid       = 4 [(gogoproto.moretags) = "json:\"TLSCertCID\" yaml:\"TLSCertCID\""];
   string type               = 5 [(gogoproto.moretags) = "json:\"type\" yaml:\"type\""];
   string version            = 6 [(gogoproto.moretags) = "json:\"version\" yaml:\"version\""];
+}
+
+message GitRepository{
+  string name =1  [(gogoproto.moretags) = "json:\"name\" yaml:\"name\""];
+  string repo_reference=2 [(gogoproto.moretags) = "json:\"repo_reference\" yaml:\"repo_reference\""];
+  string description=3  [(gogoproto.moretags) = "json:\"description\" yaml:\"description\""];
+  string version            = 4 [(gogoproto.moretags) = "json:\"version\" yaml:\"version\""];
+}
+
+message Binary{
+  string raw_multicodec=1;
```

Comment:
This was underspecified on my part, instead of a `raw_multicodec` field and a `raw_binary` field, all we need is a `hash_reference` field. And this reference will be a CID or multihash which uses the [raw mutliformat prefix](https://github.com/multiformats/multicodec/blob/master/table.csv#L40) and references the binary as  an IPLD object.